### ISSUE TITLE
Fix false positives for :autofill in selector-pseudo-class-no-unknown

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#5168]
> Clearly describe the bug

Firefox 86 supports the `:autofill` pseudo-class.
https://hacks.mozilla.org/2021/02/a-fabulous-february-firefox-86/

> Which rule, if any, is the bug related to?

`selector-pseudo-class-no-unknown`

> What code is needed to reproduce the bug?

```css
:autofill {
  border: 3px solid blue;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "selector-pseudo-class-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

https://stylelint.io/demo
